app/build.gradle

```diff
@@ -24,8 +24,10 @@ android {
 }
 
 dependencies {
+
+    implementation project(":domain")
     implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
put it, at the beginning of dependencies block

Reply:
please sort dependencies 
 
`implementation fileTree(dir: 'libs', include: ['*.jar'])`
`  implementation project(":domain")`

Reply:
Done